pokedex/src/Components/App/App.jsx

```diff
@@ -0,0 +1,141 @@
+
+const App = () => {
+  const [loading, setLoading] = useState(true);
+  const [url, setUrl] = useState("https://pokeapi.co/api/v2/pokemon/");
+  const [activePokemon, setActivePokemon] = useState();
+  const [pokeInfo, setPokeInfo] = useState({});
+  const [searchTerm, setSearchTerm] = useState("");
+
+  const [pokemonList, setPokemonList] = useState(pokemonData.results);
+  const filteredPokemonList = pokemonList.filter((pokemon) =>
+    pokemon.name.includes(searchTerm)
+  );
+
+  const pokeData = Object.values(pokeInfo).filter((pokemon) => {
+    const bool = pokemon.name.includes(searchTerm);
+    return bool;
+  });
+
+  const navigate = useNavigate();
+
+  const pokeFun = async () => {
+    setLoading(true);
+    const res = await axios.get(url);
+    fetchPokemonDetails(res.data.results);
+    setLoading(false);
+  };
+
+  async function getPokemonDetails(url) {
+    const pokemon = await axios.get(url);
+    setActivePokemon(pokemon.data);
+  }
+
+  const fetchPokemonDetails = async (res) => {
+    res.map(async (item) => {
+      const result = await axios.get(item.url);
+
+      setPokeInfo((state) => {
+        state = { ...state, [result.data.id]: result.data };
+        return state;
+      });
+    });
+  };
+
+  useEffect(() => {
+    pokeFun();
+  }, [url]);
+
+  return (
+    <div className="container">
+      <Routes>
+        {
+          <Route
+            exact
+            path="/"
+            element={
+              <>
+                <Header
+                  pokemonList={pokemonList}
+                  filteredPokemonList={filteredPokemonList}
+                  searchTerm={searchTerm}
+                  setSearchTerm={setSearchTerm}
+                />
+                <ul>
+                  {filteredPokemonList.map((pokemon) => (
+                    <li key={pokemon.id} className="pokemon-item">
+                      <a
+                        onClick={() => {
+                          getPokemonDetails(pokemon.url);
+                          navigate("/pokemon");
+                        }}
+                      >
+                        {pokemon.name}
+                      </a>
+                    </li>
+                  ))}
+                </ul>
+              </>
+            }
+          />
+        }
+
+        <Route
+          path="/pokemon"
+          element={
+            <>
+              <Header
+                pokemonList={pokemonList}
+                filteredPokemonList={filteredPokemonList}
+                searchTerm={searchTerm}
+                setSearchTerm={setSearchTerm}
+              />
+              <PokemonCards
+                pokeData={pokeData}
+                loading={loading}
+                setActivePokemon={setActivePokemon}
+                setPokeInfo={setPokeInfo}
+                setUrl={setUrl}
+                activePokemon={activePokemon}
+              />
+            </>
+          }
+        />
+        <Route path="/aboutme" element={<Aboutme />} />
```

Comment:
The `About me` navigation link is not visible on the page

Reply:
I put the pokeball icon that leads to the About me page